pkgs/development/compilers/cudatoolkit/common.nix

```diff
@@ -119,6 +119,12 @@ stdenv.mkDerivation rec {
         fi
       done
       mv pkg/builds/cuda_nvcc/nvvm $out/nvvm
+
+      mv pkg/builds/cuda_sanitizer_api $out/cuda_sanitizer_api
+      ln -s $out/cuda_sanitizer_api/compute-sanitizer/compute-sanitizer $out/bin/compute-sanitizer
+
+      mv pkg/builds/nsight_systems/target-linux-x64 $out/target-linux-x64
+      mv pkg/builds/nsight_systems/host-linux-x64 $out/host-linux-x64
```

Comment:
Are compute-sanitizer and nsys only available for cuda 11.x? It looks like nsys is at least [compatible with 10.x](https://docs.nvidia.com/nsight-systems/InstallationGuide/index.html#cuda-support) as well. I couldn't find docs on compute-sanitizer.

In the long run we should be getting rid of cudatoolkit_10 anyways, so if it's tricky to support previous versions than no need to waste time on it.

Reply:
Thanks, I think you're right that we can support nsys for >=10.0 and [compute-sanitizer for >=10.1)(https://docs.nvidia.com/compute-sanitizer/ReleaseNotes/index.html#updates-2019-1).

Reply:
The package structure is a bit different in 10.x, so it doesn't seem like a simple matter of changing the `versionAtLeast` bound. For now I'll focus on getting the compute-sanitizer binary working as expected (without needing to set `LD_LIBRARY_PATH`)

Reply:
Ok, no need to worry about it then!